ods_ci/utils/scripts/testconfig/test-variables.yml

```diff
@@ -81,3 +81,4 @@ APPLICATIONS_NAMESPACE: APPLICATIONS_NAMESPACE
 MONITORING_NAMESPACE: MONITORING_NAMESPACE
 OPERATOR_NAMESPACE: OPERATOR_NAMESPACE
 NOTEBOOKS_NAMESPACE: NOTEBOOKS_NAMESPACE
+OPENSHIFT_PIPELINES_VERSION: OPENSHIFT_PIPELINES_VERSION
```

Comment:
where do you define the default value for `OPENSHIFT_PIPELINES_VERSION`?

Reply:
In a private repository. Going to share it with you